### PR TITLE
From feature request #17314: add support to enable kinit options [-f|-F] and [-a|-A]

### PR DESCRIPTION
**Pull request by kiwy42**
Add 2 default configuration value:
[kerberos]
forwardable = True (this one will trigger kinit argument -f or -F)
include_ip = True ( this one will trigger kinit argument -a or -A)

Edit kerberos.py to enable usage of new argument [-f | -F] and [-a | -A]

I don't know if this pull request is sufficiant, it's my first time contributing to such a big project and in general my second contribution all around on github.  
If any of you is able to help, it would be awesome to see this pull request integrated however I do not know how to write test or more documentation. any help is welcome.

### CODE REVIEW

**Comment by potiuk** [comment]
If all the tests pass, I think the change should be safe - but this might be a good opportunity to live the world a little better after the change than it was before :).

Happy to guide you. I think we do not have (historically) a lot of tests about kerberos, but this might be a nice opportunity to add some unit tests to "renew_kt" function. Also it might be a nice opportunity to review and improve kerberos integration documentation - which might need some polishing (or even adding).

 Would you like to try ?

It is a bit involved and requires to learn about mocking (but I can point you to a few examples that you can learn from).

**Comment by kiwy42** [comment]
> 
> 
> If all the tests pass, I think the change should be safe - but this might be a good opportunity to live the world a little better after the change than it was before :).
> 
> Happy to guide you. I think we do not have (historically) a lot of tests about kerberos, but this might be a nice opportunity to add some unit tests to "renew_kt" function. Also it might be a nice opportunity to review and improve kerberos integration documentation - which might need some polishing (or even adding).
> 
> Would you like to try ?
> 
> It is a bit involved and requires to learn about mocking (but I can point you to a few examples that you can learn from).

Amazing reply, at the moment I'm in the rush to finish my airflow integration at work and can work with my custom builded version, when things get smoother I'd be delighted to help in incorporating a bit more than just that.  
Howerver cleary I'm not a very good Python writer (in fact I really do not like Python as a langage) and not really confortable with continus integration  and unit testing in Python. So I have a lot to work on before being able to improve kerberos integration.  
I'll considering helping anyway.

**Review by subkanthi — APPROVED** [review]

**Comment by potiuk** [comment]
No worries - next release is few weeks away. We have time to iterate. Happy to help when you will have time

**Comment by potiuk** [comment]
Just rebase from time-to time and add tests as-you-go.

**Comment by kiwy42** [comment]
I have tried to correct my typo in `kerberos.py` however I'm so lost with  the pull request workflow that I do not know if I did anything wrong.   
Any hint would be greatly appreciated. 
Thank you

File: docker-context-files/README.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
-<!--
```
Comment (potiuk) [inline docker-context-files/README.md:1]: Do not remove this file please.

**Review by potiuk — CHANGES_REQUESTED** [review]
Can you please rebase and restore the README file?

**Comment by kiwy42** [comment]
Sure enough, I will monday retry this.
It's my very first attempt at participating in such a big product and I'm not enough skilled with `git` I'll redo everything and push it again. 
I'm currently using those modification in production and they seems OK so far.

**Comment by kiwy42** [comment]
So I'm quiet bad when it come to git so I destroy my repo and reforked and add my changes and my 3 changes only in this repo: https://github.com/kiwy42/airflow However it seems I've broken the link between this pull request and my repo which has been destroyed and reforked.  
Should I reissue a pull request ? I'm not sure to understand everything here.

**Comment by potiuk** [comment]
Just close the PR and open a new one

**Pull request closed without merging**